src/github.js

```diff
@@ -199,21 +197,21 @@ ns.bumpFiles = function (bump) {
     var v = versiony.from('package.json').indent(jsonIndent);
 
     switch (bump) {
-        case 'major':
-            v.newMajor();
-            break;
+    case 'major':
```

Comment:
Looks like we can configure that (if the team agrees with me, which I don't think they do):
http://eslint.org/docs/rules/indent#options